packages/next-steps/index.tsx

```diff
@@ -89,7 +110,10 @@ function Tile({ theme, variant, heading, description, cta }) {
         }}
       >
         <div className={s.tileInner}>
-          <h3 className={s.tileHeading}>{heading}</h3>
+          <div className={s.tileHeadingWrapper}>
+            <h3 className={s.tileHeading}>{heading}</h3>
+            {badge ? <BadgeWrapper theme={theme}>{badge}</BadgeWrapper> : null}
```

Comment:
Were we able to keep the badge inline with the text when wrapping? Or what is the expected behavior here?

<img width="921" alt="CleanShot 2023-01-18 at 15 05 20@2x" src="https://user-images.githubusercontent.com/825855/213283415-2ac577df-2491-4b97-9b14-852acbd94462.png">

Reply:
Oops - I think you flagged this before. Think this does the trick:

<img width="644" alt="Screen Shot 2023-01-18 at 12 32 33 PM" src="https://user-images.githubusercontent.com/39201593/213288503-2523494b-eee4-459b-afc4-363da603b577.png">